bin/fixedcase/special-case-titles

```diff
@@ -0,0 +1,98 @@
+session 10: the lexicon
+some remarks on J. L. Mey’s paper (preprint no. 20)
+speech comparison in the Rosetta Stone TM
+summary of session 10 - continous speech recognition II
+summary of session 2: spoken language systems I
+the Finite String newsletter: 1986 ACL membership list
+the Finite String newsletter: ACL 1986 annual meeting
+the Finite String newsletter: abstracts of current literature
+the Finite String newsletter: abstracts of current literature
+the Finite String newsletter: announcements
+the Finite String newsletter: author index: volumes 6-12
+the Finite String newsletter: calendar of meetings
+the Finite String newsletter: call for papers
+the Finite String newsletter: calls for papers
+the Finite String newsletter: calls for papers, proposals, pictures, for award nominations
+the Finite String newsletter: site report: linguistics in Germany
+the Finite String newsletter: title index: volumes 6-12
+the Finite String, volume 11, number 1
+the Finite String, volume 11, number 2
+the Finite String, volume 11, number 3
+the Finite String, volume 11, number 3 (continued)
+the Finite String, volume 11, number 4
+the Finite String, volume 12, number 1
+the Finite String, volume 12, number 2
+the Finite String, volume 12, number 2 (continued)
+the Finite String, volume 12, number 3
+the Finite String, volume 12, number 3, continued
+the Finite String, volume 12, number 4
+the Finite String, volume 12, number 4 (continued)
+the Finite String, volume 12, number 5
+the Finite String, volume 12, number 6
+the Finite String, volume 12, number 6 (continued)
+the Finite String, volume 13, number 1
+the Finite String, volume 13, number 1 (continued)
+the Finite String, volume 13, number 2
+the Finite String, volume 13, number 3
+the Finite String, volume 13, number 4
+the Finite String, volume 13, number 4 (continued)
+the Finite String, volume 13, number 5
+the Finite String, volume 13, number 5
+the Finite String, volume 13, number 6
+the Finite String, volume 13, number 6 (continued)
+the Finite String, volume 13, number 7
+the Finite String, volume 13, number 8
+the Finite String, volume 14, number 1
+the Finite String, volume 14, number 1 (continued)
+the Finite String, volume 14, number 1 (continued)
+the Finite String, volume 14, number 1 (continued)
+the Finite String, volume 14, number 2
+the Finite String, volume 14, number 3
+the Finite String, volume 14, number 4
+the Finite String, volume 14, number 5
+the Finite String, volume 14, number 6
+the Finite String, volume 14, number 6 (continued)
+the Finite String, volume 14, number 7
+the Finite String, volume 15, number 1
+the Finite String, volume 15, number 2
+the Finite String, volume 15, number 2 (continued)
+the Finite String, volume 15, number 3
+the Finite String, volume 15, number 4
+the Finite String, volume 15, number 5
+the Finite String, volume 15, number 5 (continued)
+three titles from the Cambridge series: studies in natural language processing
```

Comment:
Shouldn't the title of the series be capitalized?

Reply:
"Studies in Natural Language Processing"? Yeah I guess